src/features/triggers/hooks/use-node-status.ts

```
@@ -0,0 +1,43 @@
+  refreshToken,
+}: UseNodeStatusOptions) {
+  const [status, setStatus] = useState<NodeStatus>("initial");
+
+  const { messages } = useRealtime({
+    channel: manualTriggerChannel,
+    topics: [topic],
+    token: refreshToken,
+    enabled: true,
+  });
+
+  useEffect(() => {
```

Biome (2.4.15)

[error] This hook specifies more dependencies than necessary: channel, topic.

(lint/correctness/useExhaustiveDependencies)